### PR TITLE
✨ add commands to build and publish image and chart to makefile

### LINKED ISSUE #1555: feature: add commands to build and publish image and chart to makefile
### Feature Description

Need to add commands to build and publish image and chart to makefile for developers and CI/CD

### Proposed Solution

Add needed commands to makefile

### Want to contribute?

- [X] I would like to work on this issue.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by pdettori**
## Summary

add commands to build and publish image and chart to makefile

Fixes #1555

### CODE REVIEW

**Comment by ezrasilvera** [comment]
I tested and it worked for me. I'll merge but we will later need to add some paragraph in the doc on how to use it

File: Makefile [hunk 1]
```diff
@@ -12,14 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Image URL to use all building/pushing image targets
-IMG ?= ghcr.io/kubestellar/kubeflex/kubestellar-operator:0.1.0
+# Image repo/tag to use all building/pushing image targets
+DOCKER_REGISTRY ?= ghcr.io/kubestellar/kubestellar
```
[the remark this answers is not in the document]
Comment (pdettori) [reply Makefile:16]: @MikeSpreitzer  Other OCI registries will work as well. I tested with quay.io and ghcr.io and both work. If the issue is in the name `DOCKER_REGISTRY` we should rename that to `OCI_REGISTRY`

[the remark this answers is not in the document]
Comment (MikeSpreitzer) [reply Makefile:16]: Docker is not exactly the same as OCI. Perhaps something simple and generic, like `CONTAINER_REGISTRY` or `IMAGE_REGISTRY` or `CONTAINER_IMAGE_REGISRY`? `OCI_REGISTRY` would not be bad.

[the remark this answers is not in the document]
Comment (pdettori) [reply Makefile:16]: `CONTAINER_REGISTRY` sounds good to me

**Pull request merged**